xml/depl_inst_nodes.xml

```diff
@@ -20,8 +20,7 @@
 <para>
   The &ostack; nodes represent the actual cloud infrastructure. Node
   installation and service deployment is done automatically from the
-  &admserv;. Before deploying the &ostack; services, &cloudos; will be
```

Comment:
For once, I think passive voice is/was appropriate here because the operator does not perform the installation process himself/herself. The operator needs to boot the nodes via PXE and allocate them in Crowbar. This will trigger the installation process which is then performed automatically.